__tests__/app.test.js

```diff
@@ -405,6 +405,41 @@ describe('/api/articles/:article_id/comments', () => {
   });
 });
 
+describe('/api/comments/:comment_id', () => {
+  describe('DELETE', () => {
+    it(`responds to a valid request with a 204 status code and successfully deletes the comment from the database`, () => {
+      return request(app)
+        .delete('/api/comments/2')
+        .expect(204)
+        .then(() => {
+          return db.query(`SELECT * FROM comments WHERE comment_id = 2`);
```

Comment:
I understand why you have this check but we don't really need to additionally check this so long as our logic in the model is only returning to the controller _if_ the comment has been deleted.  The fact that we reject the promise if rows.length is falsy is enough to do this.